src/thermo/Elements.cpp

```diff
@@ -192,7 +238,13 @@ double getElementWeight(int atomicNumber)
     if (atomicNumber > num || atomicNumber < 1) {
         throw IndexError("getElementWeight", "atomicWeightTable", atomicNumber, num);
     }
-    return atomicWeightTable[atomicNumber - 1].atomicWeight;
+    double elementWeight = atomicWeightTable[atomicNumber - 1].atomicWeight;
+    if (elementWeight < 0.0) {
+        string ename = getElementName(atomicNumber);
+        throw CanteraError("getElementWeight",
+            "element has no stable isotopes: " + ename);
```

Comment:
```suggestion
            "element '{}' has no stable isotopes: ", getElementName(atomicNumber));
```
and eliminate the variable `ename`.